e2e-tests/README.md

```diff
@@ -0,0 +1,103 @@
+# Building and testing the Operator
+
+## Requirements
+
+You need to install a number of software packages on your system to satisfy the build dependencies for building the Operator and/or to run its automated tests.
+
+### CentOS
+
+```
+sudo yum -y install epel-release https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+sudo yum -y install coreutils sed jq curl yq
+curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
+    | tar -C /usr/bin --strip-components 1 --wildcards -zxvpf - '*/oc' '*/kubectl'
+curl -s https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz \
+    | tar -C /usr/bin --strip-components 1 -zxvpf - '*/helm'
+curl https://sdk.cloud.google.com | bash
+```
+
+### MacOS
+
+```
+brew install coreutils gnu-sed jq kubernetes-cli openshift-cli kubernetes-helm yq
```

Comment:
yq v3 should be installed now , we need to mention it because v4 does not work with our code